### ISSUE TITLE
team colors in hashketball

### ISSUE THREAD

**Issue by dougtebay** [learn-co-curriculum/advanced-hashes-hashketball#14]
Hi, just a note that the Readme for hashketball calls for lowercase colors, but the test requires uppercase for #team_colors to pass.

**Comment by AnnJohn** [comment]
Thanks @dougtebay for raising this issue. Could you let us know where exactly in the Readme it calls for lowercase letters? A screenshot would be helpful. Going through the Readme, I saw that the colors they called for were in uppercase letters but I might be missing something. (I posted an example of one below.)Thank again!

![colors](https://cloud.githubusercontent.com/assets/1892150/9580089/3925c076-4fc5-11e5-896d-9d5fb1c1049e.png)

**Comment by dougtebay** [comment]
Sure, below is what I see. Based on the difference between your screen cap and mine we seem to be looking at different readmes. I think there's some issue with multiple versions of the hasketball lesson. When I first started the lesson the "You forked this lab." link worked. When I was ready to submit it I followed the link again and got a 404--which I still get (https://github.com/dougtebay/hashketball-001-prework-web). This is the only lab where I ran into this kind of problem.
![screen shot 2015-08-31 at 12 08 03 pm](https://cloud.githubusercontent.com/assets/13855895/9583415/3182b20c-4fd9-11e5-940f-8a38b8174c17.png)
